bundles/org.openhab.binding.magentatv/src/main/java/org/openhab/binding/magentatv/internal/MagentaTVHandler.java

```diff
@@ -0,0 +1,575 @@
+                    }
+                }
+
+                connectReceiver(); // throws MagentaTVException on error
+                // change to ThingStatus.ONLINE will be done when the pairing result is received
+                // (see onPairingResult())
+            } catch (MagentaTVException | InterruptedException | RuntimeException e) {
+                errorMessage = e.getMessage() != null ? e.getMessage() : e.getClass().toString();
+            } finally {
+                if (!errorMessage.isEmpty()) {
+                    setOnlineState(ThingStatus.OFFLINE, errorMessage);
+                }
+            }
+        });
+    }
+
+    /**
+     * Handle channel commands
+     *
+     * @param channelUID - the channel, which received the command
+     * @param command - the actual command (could be instance of StringType,
+     *            DecimalType or OnOffType)
+     */
+    @SuppressWarnings("null")
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+
+        if (command == RefreshType.REFRESH) {
+            // currently no channels to be refreshed
+            return;
+        }
+
+        try {
+            if (!isOnline() || command.toString().equalsIgnoreCase("PAIR")) {
+                logger.info("Device {0} is offline, try to (re-)connect", deviceName());
+                connectReceiver(); // reconnect to MR, throws an exception if this fails
+            }
+
+            logger.debug("Channel command for device {0}: {1} for channel {2}", thingConfig.getFriendlyName(),
+                    command.toString(), channelUID.getId().toString());
+            switch (channelUID.getId()) {
+                case CHANNEL_POWER: // toggle power
+                    logger.info("Toggle power, new state={0}", command.toString());
+                    control.sendKey("POWER");
+                    break;
+                case CHANNEL_CHANNEL:
+                    control.selectChannel(command.toString());
+                    break;
+                case CHANNEL_CHUP:
+                    control.sendKey("CHUP");
+                    updateState(CHANNEL_CHUP, OnOffType.OFF);
+                    break;
+                case CHANNEL_CHDOWN:
+                    control.sendKey("CHDOWN");
+                    updateState(CHANNEL_CHDOWN, OnOffType.OFF);
+                    break;
+                case CHANNEL_VOLUP:
+                    control.sendKey("VOLUP");
+                    updateState(CHANNEL_VOLUP, OnOffType.OFF);
+                    break;
+                case CHANNEL_VOLDOWN:
+                    control.sendKey("VOLDOWN");
+                    updateState(CHANNEL_VOLDOWN, OnOffType.OFF);
+                    break;
+                case CHANNEL_KEY:
+                    if (command.toString().equalsIgnoreCase("PAIR")) { // special key to re-pair receiver
+                        logger.info("PAIRing key received, reconnect device {0}", deviceName());
```

Comment:
change to logger .debug

Reply:
want to keep this info. It indicates some trouble in communication with the MR if users need to use the PAIRing key. So it they report it in a community thread it's a clear indicator.